.pre-commit-config.yaml

```diff
@@ -0,0 +1,73 @@
+---
+files: ^cli/
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.10
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: check-case-conflict
+      - id: detect-private-key
+      - id: check-ast
+
+  - repo: https://github.com/psf/black
+    rev: 21.7b0
+    hooks:
+      - id: black
+        args: ['--line-length', '110']
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.9.2
+    hooks:
+      - id: isort
+        args:
+        - --profile=black
+        - --color
+        - --line-length=110
+
+  - repo: https://gitlab.com/PyCQA/flake8
+    rev: '3.9.2'
+    hooks:
+      - id: flake8
+        additional_dependencies: ['pep8-naming', 'flake8-docstrings']
+
+  - repo: https://github.com/PyCQA/pylint
+    rev: v2.9.5
+    hooks:
+      - id: pylint
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+      - id: mypy
+        args:
+        - --ignore-missing-imports
+        - --install-types
+        - --non-interactive
+
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.21.2
+    hooks:
+      - id: pyupgrade
+
+  - repo: https://github.com/PyCQA/bandit
+    rev: 1.7.0
+    hooks:
+      - id: bandit
+        args: ["--exclude", "cli/tests/", "--skip", "B404,B603"]
```

Comment:
I think that this exclude might be leftover from snafu's pre-commit config? Might be ok to chuck it

Reply:
The tests uses asserts which made bandit fail but I didn't want to remove skip the rule entirely since I want it to fail if asserts are in the actual code base

Reply:
My bad on these `cli/tests` path comments that I left. I need to go get a pair of reading glasses or something- I mistook that for `ci/tests`. Please ignore, my bad :+1: